src/calculateRentalCost.js

```diff
@@ -27,8 +27,22 @@
  * @return {number}
  */
 
+const DAILY_PRICE = 40;
+const SMALL_DISCOUNT = 20;
+const BIG_DISCOUNT = 50;
+const SHORT_TERM = 3;
+const LONG_TERM = 7;
+
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE = days * DAILY_PRICE;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return BASE_PRICE - SMALL_DISCOUNT;
+  } else if (days >= LONG_TERM) {
+    return BASE_PRICE - BIG_DISCOUNT;
+  }
```

Comment:
checklist point 5 - you don't need to use `else if` when you have `return` inside an `if` statement :)